### DISCUSSION TITLE
Bot is not working at all

### DISCUSSION THREAD

**Discussion by Ipsi-SG** [pallupz/covid-vaccine-booking#204] [Q&A]
the was working till 23rd from 24th It is not working at all.
Getting timeout. Please help.

**Comment by akuma2519** [comment]
even i am getting the below error from yesterday night::

Successfully requested OTP for mobile number 998069Xxx1 at 2021-05-25 10:42:48.652188..
Enter OTP (If this takes more than 2 minutes, press Enter to retry): 175961
Validating OTP..
Unable to Validate OTP
Response: {"message": "Endpoint request timed out"}
Retry with 9980699111 ? (y/n Default y): y
Successfully requested OTP for mobile number 9980699111 at 2021-05-25 10:44:45.844968..
Enter OTP (If this takes more than 2 minutes, press Enter to retry): 175961
Validating OTP..
Unable to Validate OTP
Response: {"message": "Endpoint request timed out"}

**Reply by pallupz** [reply]
this issue is resolved in the latest code

**Reply by Ipsi-SG** [reply]
Working. But getting timeout very soon.
Another error.
Retry with 9742951791 ? (y/n Default y): y
Unable to Generate OTP
429 {
 "error": "Too Many Requests"
}
